docs/design/runtime_processing/runtime_recording.md

```diff
@@ -164,17 +163,17 @@ string select_nodes  # reserved
 string select_topics # reserved
 ```
 
-The recording_frequency is the frequency at which each process records initialization trace data in the PREPARE state.
+CARET records sets of meta-information to a LTTng ring-buffer one by one rather than tries to store those meta-information at once.
+CARET serves a parameter, `recording_frequency`, to control velocity to record meta-information.
+`recording_frequency` is frequency at which each process records meta-information. It decides how many sets of meta-information is stored to the ring-buffer per second.
 The higher the frequency, the shorter the time required to complete the PREPARE state, but the greater the risk of tracer discarded.
 
 `ignore_nodes` `ignore_topics` `select_nodes` `select_topics` are unimplemented features.
 They are reserved fields for setting [tracepoint filtering](./tracepoint_filtering.md) at the start of the measurement from CLI.
 
 <prettier-ignore-start>
 !!!Info
-    As another method of recording_frequency,
-    initialization-related trace points can be recorded in [blocking mode](https://lttng.org/blog/2017/11/22/lttng-ust-blocking-mode/) to prevent tracer discarded.
-    This method reduces the occurrence of data loss, but this time we decided to improve it on the caret side to reduce implementation changes.
+    Another method to avoid tracer discarded is writing meta-information with [blocking mode](https://lttng.org/blog/2017/11/22/lttng-ust-blocking-mode/). LTTng serves a function to apply blocking mode to chosen events, and chosen events are written to disks exactly. Blocking mode will reduce occurrence of data loss. In this moment, `recording_frequency` is introduced to mitigate data loss because range of influence is smaller than blocking mode.
```

Comment:
```suggestion
    Another method to avoid tracer discarded is writing meta-information with [blocking mode](https://lttng.org/blog/2017/11/22/lttng-ust-blocking-mode/). LTTng serves a function to apply blocking mode to chosen events, and chosen events are written to disks exactly. Blocking mode will reduce occurrence of data loss. In this moment, `recording_frequency` is introduced to mitigate data loss because range of influence on implementation is smaller than blocking mode.
```